gallery/src/components/NxTag/NxTagPage.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import {GalleryDescriptionTile, GalleryExampleTile} from '../../gallery-components/GalleryTiles';
+import NxTagExample from './NxTagExample';
+import NxSelectableTagExample from './NxSelectableTagExample';
+import NxTagNarrowExample from './NxTagNarrowExample';
+import { NxTable, NxTableHead, NxTableCell, NxTableRow, NxTableBody } from '@sonatype/react-shared-components';
+
+const NxTagExampleCode = require('!!raw-loader!./NxTagExample').default;
+const NxSelectableTagExampleCode = require('!!raw-loader!./NxSelectableTagExample').default;
+const NxTagNarrowExampleCode = require('!!raw-loader!./NxTagNarrowExample').default;
+
+const NxTagPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        There are two types of <code className="nx-code">NxTag</code>. The basic tags have a single state.
+        <code className="nx-code">NxSelectableTag</code> can be selected and deselected. It is up to the consumer
+        what that means in the context of the UI. The are many colors provided for
+        <code className="nx-code">NxTag</code> which can be specified via props.
```

Comment:
Fix missing whitespace
```suggestion
        what that means in the context of the UI. The are many colors provided
        for <code className="nx-code">NxTag</code> which can be specified via props.
```